src/slave_node/measuring_node.py

```diff
@@ -0,0 +1,37 @@
+import canopen
+import logging
+import yaml
+from time import sleep
+
+'''
+    We are not going to use this for the project but can be
+    used for testing purposes.
+'''
+
+
+class MeasuringSlaveNode:
+    def __init__(self):
+        with open('../config.yaml', 'r') as ymlfile:
+            self.config = yaml.safe_load(ymlfile)
+        # local node
+        self.connectToNetwork(canopen.Network())
+        self.network.create_node(1, '../eds_files/measuring_distance_node.eds')
+        while True:
+            sleep(1)
+            print(self.network[1].sdo[0x2000].phys)
+            self.network[1].sdo.download(0x2000, 0, bytes(
+                [self.network[1].sdo[0x2000].phys + 1, 0]))
+            self.network.sync.transmit()
+
+    def connectToNetwork(self, network):
+        self.network = network
+        try:
+            self.network.connect(bustype=self.config['canopen_vcan']['bustype'],
```

Comment:
Is het niet handig om die 'canopen_vcan' als variabel in de code te zetten? Dan kan je een wegcommenten als je een ander wil gebruiken.
Als je dan van interface switcht, hoef je niet 3 lines aan te passen

Reply:
Dus bijvoorbeeld bij line 11:

```
INTERFACE = 'canopen_vcan'
#INTERFACE = 'canopen_slcan'
```
dan line 29:
`self.network.connect(bustype=self.config[INTERFACE]['bustype'],`

Zodoende hoef je alleen weg te commenten welke interface je niet wilt gebruiken